src/app/attendances/page.tsx

```diff
@@ -0,0 +1,14 @@
+import TopNavBar from '../molecules/top-navigation-bar';
+import AttendanceStatus from '../molecules/attendance-status';
+import SubjectAttend from '../organisms/subject-attend';
+
+const AttendancesPage = () => {
+  return (
+    <main className={styles.main}>
```

Comment:
```suggestion
    <main>
```
この `styles` は存在しません.